study/src/test/java/reflection/Junit4TestRunner.java

```diff
@@ -9,5 +10,12 @@ void run() throws Exception {
         Class<Junit4Test> clazz = Junit4Test.class;
 
         // TODO Junit4Test에서 @MyTest 애노테이션이 있는 메소드 실행
+        for (Method method : clazz.getDeclaredMethods()) {
+            MyTest annotation = method.getAnnotation(MyTest.class);
+            if(annotation != null) {
```

Comment:
```suggestion
            if (method.isAnnotationPresent(MyTest.class) {}
```

AnnotationHandler에서는 RequestMapping 어노테이션을 사용하니 get으로 가져온 뒤 null 확인을 하는 것이 필요하지만, 여기서는 이 어노테이션이 안 쓰이니 present만 확인하면 될 것 같네요~

Reply:
동의합니다:) 이 처리가 더 나아보이네요~!
감사합니다!